at_utils/lib/src/logging/handlers.dart

```diff
@@ -8,6 +8,15 @@ abstract class LoggingHandler {
 }
 
 class ConsoleLoggingHandler implements LoggingHandler {
+  static final ConsoleLoggingHandler _singleton =
```

Comment:
Doesn't need to be a forced singleton. Can just be a normal class. The change you've made to have a static instance in atsignlogger.dart is sufficient